resources/i18n/fr.json

```diff
@@ -0,0 +1,108 @@
+{
+    "@metadata": {
+		"authors": [
+		]
+	},
+    "name":"Français",
+    "error-downloader-window-title":"Impossible de lancer le system de téléchargement",
+    "error-downloader-launch-message":"Impossible de lancer le system de téléchargement, Kiwix-desktop fonctionnera mais sans les fonctionnalités de téléchargement !",
```

Comment:
small error `system` instead of `système`

Reply:
Fixed